sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v2/checks.sql

```diff
@@ -0,0 +1,32 @@
+#fail
+{{ is_unique(columns=["client_id"]) }}
+
+#fail
```

Comment:
Should this be a fail and stop ETL execution?

Reply:
Yeah if there's a missing client something has gone wrong